files-community/Files
# Fix: Fixed an issue where selection would get canceled after pressing upper arrow key

**Resolved / Related Issues**
   Closes #13798 

**Validation**
How did you test these changes?
   1. Open app
   2. Select an item
   3. Press F2 to rename
   4. Move pointer to middle of name
   5. Press Shift+UpArrow many times
   6. Confirm that selection does not get canceled
   7. Go to step 4
   8. Press Shift+DownArrow many times
   9. Confirm that selection does not get canceled
